website/docs/components/time/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Time
+description: The Time component is used to display dates and time in a consistent format.
+caption: Displays dates and times in a consistent format.
+related: ['components/form/text-input']
+links:
+  figma: {Link}
+  github: https://github.com/hashicorp/design-system/tree/main/packages/components/src/components/hds/time
+previewImage: assets/illustrations/components/time.jpg
+navigation:
+  keywords: ['time']
+---
+
+<section data-tab="Guidelines">
+  @include "partials/guidelines/overview.md"
+  @include "partials/guidelines/guidelines.md"
+</section>
+
+<section data-tab="Code">
+  @include "partials/code/how-to-use.md"
+  @include "partials/code/component-api.md"
+</section>
+
+<section data-tab="Specifications">
+  @include "partials/specifications/anatomy.md"
+  @include "partials/specifications/states.md"
+</section>
+
+<section data-tab="Accessibility">
```

Comment:
Should we add a version history to new components? seems like were moving towards all component pages having one, I guess for this one it would just have the version number and say that is when it was released.

Reply:
We don't seem to have done that previously but I don't know if we should or not.

Any other opinions on whether we should start doing this? @alex-ju @majedelass

Reply:
It seem like we did it for the `AppHeader` https://github.com/hashicorp/design-system/blob/main/website/docs/components/app-header/partials/version-history/4.8.0.md#480 – I believe it's worth having it for any type of change (add, update, deprecate)

Reply:
Added version history with release.